apps/web/pages/[user].tsx

```diff
@@ -179,50 +239,53 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   const web3Credentials = credentials.find((credential) => credential.type.includes("_web3"));
 
-  const eventTypesWithHidden = await prisma.eventType.findMany({
-    where: {
-      AND: [
-        {
-          teamId: null,
-        },
-        {
-          OR: [
+  const eventTypesWithHidden =
+    usernameList.length > 1
+      ? []
+      : await prisma.eventType.findMany({
+          where: {
+            AND: [
+              {
+                teamId: null,
+              },
+              {
+                OR: [
+                  {
+                    userId: users[0].id,
+                  },
+                  {
+                    users: {
+                      some: {
+                        id: users[0].id,
+                      },
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+          orderBy: [
             {
-              userId: user.id,
+              position: "desc",
             },
             {
-              users: {
-                some: {
-                  id: user.id,
-                },
-              },
+              id: "asc",
             },
           ],
-        },
-      ],
-    },
-    orderBy: [
-      {
-        position: "desc",
-      },
-      {
-        id: "asc",
-      },
-    ],
-    select: {
-      id: true,
-      slug: true,
-      title: true,
-      length: true,
-      description: true,
-      hidden: true,
-      schedulingType: true,
-      price: true,
-      currency: true,
-      metadata: true,
-    },
-    take: user.plan === "FREE" ? 1 : undefined,
-  });
+          select: {
+            id: true,
+            slug: true,
+            title: true,
+            length: true,
+            description: true,
+            hidden: true,
+            schedulingType: true,
+            price: true,
+            currency: true,
+            metadata: true,
+          },
+          take: users[0].plan === "FREE" ? 1 : undefined,
```

Comment:
Can we use Enum here instead of "FREE"

Reply:
I believe this is how we are using it everywhere. TypeScript gives us type safety so the wrong value can't be used.